front/components/assistant/AssistantDetails.tsx

```diff
@@ -121,6 +165,70 @@ export function AssistantDetails({
       "This assistant has no instructions."
     );
 
+  const FeedbacksSection = () => {
+    const {
+      agentConfigurationFeedbacks,
+      isAgentConfigurationFeedbacksLoading,
+    } = useAgentConfigurationFeedbacks({
+      workspaceId: owner.sId,
+      agentConfigurationId: assistantId ?? "",
+    });
+
+    const sortedFeedbacks = useMemo(() => {
+      if (!agentConfigurationFeedbacks) {
+        return null;
+      }
+      return agentConfigurationFeedbacks.sort(
+        (a, b) =>
+          new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime()
+      );
+    }, [agentConfigurationFeedbacks]);
+
+    const { agentConfigurationHistory, isAgentConfigurationHistoryLoading } =
+      useAgentConfigurationHistory({
+        workspaceId: owner.sId,
+        agentConfigurationId: assistantId || "",
+        disabled: !assistantId,
+      });
+
+    return isAgentConfigurationFeedbacksLoading ||
+      isAgentConfigurationHistoryLoading ? (
+      <Spinner />
+    ) : (
+      <div>
+        {!sortedFeedbacks || sortedFeedbacks.length === 0 || !assistantId ? (
+          <div className="mt-3 text-sm text-element-900">No feedbacks.</div>
+        ) : (
+          <div className="mt-3">
+            <ConfigVersionHeader
```

Comment:
Let's not use weird shortcuts names we don't use anywhere -> `AgentConfigurationVersionHeader`